test/integration/live-api.test.js

```diff
@@ -104,8 +101,6 @@ const genericOptions = {
   waitForCompletion: true,
 }
 
-jest.setTimeout(100000)
```

Comment:
what's the timeout in vitest? i believe this timeout was added because jest's default timeout was too low. unless vitest default timeout is at least 100 sec we should probably set the timeout for vitest too

Reply:
Good catch. I added back some timeouts. The syntax for this is a bit different in Vitest.